watertap/property_models/tests/test_cryst_prop_pack.py

```diff
@@ -100,8 +95,8 @@ class TestDefaultNaClwaterProperty:
                             ('mass_frac_phase_comp', ('Vap', 'H2O')) : 1.0,
                             ('flow_mol_phase_comp', ('Liq', 'H2O')) : 53.57,
                             ('flow_mol_phase_comp', ('Liq', 'NaCl')) : 0.5989,
-                            ('flow_mol_phase_comp', ('Sol', 'NaCl')) : 3.811504179523017e-08,
-                            ('flow_mol_phase_comp', ('Vap', 'H2O')) : 3.8114854951498772e-06,
+                            ('flow_mol_phase_comp', ('Sol', 'NaCl')) : 1.6318503332497833e-09,
```

Comment:
I do need to check if this change is signficiant?

Reply:
No, this value is fine. It is basically an approximation of zero.